### PR TITLE
Fix typo in comment && document

### PR DESCRIPTION
**Pull request by ttyio**
Fix typo in comment && document

### CODE REVIEW

File: include/vulkan/vk_layer.h [hunk 1]
```diff
@@ -262,8 +262,8 @@ typedef enum VkLayerDbgAction_ {
 // CreateInstance and CreateDevice support structures
 
 /* Sub type of structure for instance and device loader ext of CreateInfo.
- * When sType == VK_STRUCTURE_TYPE_LAYER_INSTANCE_CREATE_INFO
- * or sType == VK_STRUCTURE_TYPE_LAYER_DEVICE_CREATE_INFO
+ * When sType == VK_STRUCTURE_TYPE_LOADER_INSTANCE_CREATE_INFO
+ * or sType == VK_STRUCTURE_TYPE_LOADER_DEVICE_CREATE_INFO
```
Comment (courtney-g) [inline include/vulkan/vk_layer.h]: This should be in a separate pull request from the unrelated doc change below.

Comment (ttyio) [inline include/vulkan/vk_layer.h]: thanks. I will close this one and resubmit.
